src/theme-provider.js

```diff
@@ -0,0 +1,85 @@
+import React, {PropTypes, Component} from 'react'
+import mitt from 'mitt'
+
+export const CHANNEL = '__glamorous__'
```

Comment:
@kentcdodds @kwelch (continuing from the now collapsed conversation)

I checked and [using Symbols in `context`](https://jsfiddle.net/v2fb93bm/) does not work, so this should remain a string.